django/library/jinja2/library/codebases/list.jinja

```diff
@@ -45,5 +45,12 @@
 {% endblock %}
 
 {% block js %}
+<script nonce="{{ request.csp_nonce }}" >
+    {% if language_facets %}
```

Comment:
can this be passed in as a data attribute on the sidebar div instead? theres a function `extractDataParams` to help with this on the js side

Reply:
you mean without attaching language_facets to window global variable?

Reply:
yeah like

```html
<div id="sidebar" {% if language_facets %} data-language-facets="{{ language_facets }}" {% endif %} />
```

mostly for consistency, not sure if there's any meaningful practical difference

Reply:
In what file would you do this? As I understand this, we'd need to do this in jinja, but we need the data in the vue component... I am lost here.. help, please!

Reply:
this one, the js side parses it with `extractDataParams` before mounting the component and passing the extracted stuff as props. Most mounting scripts in `/apps` do it

Reply:
oh, got it! thanks!